hack/s3_test/test1/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM public.ecr.aws/debian/debian:bullseye-slim
+
+RUN echo first
```

Comment:
These commands should write a constant plus a random value. See https://github.com/moby/buildkit/blob/master/client/client_test.go#L1410 . Then when you build them export the result to a local directory and check that the unique part has not change. Therefore cache matched.

Reply:
Done